python/ray/data/read_api.py

```diff
@@ -1755,6 +1756,114 @@ def create_connection():
     )
 
 
+@PublicAPI(stability="alpha")
+def read_databricks_tables(
+    *,
+    warehouse_id: str,
+    table: Optional[str] = None,
+    query: Optional[str] = None,
+    catalog: Optional[str] = None,
+    schema: Optional[str] = None,
+    parallelism: int = -1,
+    ray_remote_args: Optional[Dict[str, Any]] = None,
+) -> Dataset:
+    """
+    Read from a Databricks unity catalog table or Databricks SQL execution result that
+    queries from Databricks UC tables.
+    Before calling `read_databricks_tables`, set the 'DATABRICKS_HOST' environment
+    variable, if this API isn't called in Databricks runtime, set the
+    'DATABRICKS_TOKEN' environment variable as well.
+
+    This reader is implemented based on
+    [Databricks statement execution API](https://docs.databricks.com/api/workspace/statementexecution).
+
+    Examples:
+    .. testcode::
+        :skipif: True
+
+        import ray
+
+        ds = ray.data.read_databricks_tables(
+            warehouse_id='a885ad08b64951ad',
+            catalog='catalog_1',
+            schema='db_1',
+            query='select id from table_1 limit 750000',
+        )
```

Comment:
In addition to this example here, could you update the "Databricks" section of "loading-data.rst"?

Reply:
Added! :)